gapis/shadertools/shadertools.go

```diff
@@ -190,7 +191,7 @@ func descriptorBindingLess(a DescriptorBinding, b DescriptorBinding) bool {
 	return a.SpirvId < b.SpirvId
 }
 
-func ExtractDebugSource(shader []uint32) (string, error) {
+func ExtractDebugSource(shader []uint32) (string, string, bool, error) {
```

Comment:
Please add a doc string comment to this function and say what all the returned values are.